sns/cm/repair/ut/cp_common.c

```diff
@@ -147,7 +147,8 @@ void cp_prepare(struct m0_cm_cp *cp, struct m0_net_buffer *buf,
 	cp->c_ag->cag_cm = cm;
 	if (!is_acc_cp)
 		cp->c_ops = &m0_sns_cm_repair_cp_ops;
-	cp->c_ops = &m0_sns_cm_acc_cp_ops;
+	else
+		cp->c_ops = &m0_sns_cm_acc_cp_ops;
```

Comment:
Did you run SNS UT/ST after this change?

Reply:
https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/1373/